src/js/chartAuthoring.js

```diff
@@ -237,6 +334,24 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
             $(this).find(dataEntryLabelSelector).val(currentData.label).trigger("change");
             $(this).find(dataEntryValueSelector).val(currentData.value).trigger("change");
         });
+
+        that.events.onDataAppliedToDataEntryPanel.fire();
+    };
+
+    floe.chartAuthoring.addSonificationControlsHandlers = function(that) {
+        var playButton = that.locate("sonifierPlay"),
+            stopButton = that.locate("sonifierStop");
+
+        playButton.click(function(e) {
```

Comment:
@amb26 I took the same approach that @cindyli recommended previously for the `bindResetClick` listener. Binding the events now looks like this and doesn't require a separate function:

```
            "onToolReady.bindPlayClick": {
                "this": "{that}.dom.sonifierPlay",
                "method": "click",
                "args": ["{chartAuthoringInterface}.sonifier.playSonification"]
            },
            "onToolReady.bindStopClick": {
                "this": "{that}.dom.sonifierStop",
                "method": "click",
                "args": ["{chartAuthoringInterface}.sonifier.stopSonification"]
            }
```

If we'd like a full declarative event binding system I think that should be a separate JIRA - in the current they aren't particularly complicated handlers & can be implemented declaratively as per above.